e2e tests NetworkPolicies are failing

**What happened**:

I wanted to run the e2e network policy tests with: `kubetest --provider=skeleton --test --test_args="--ginkgo.focus=\[Feature:NetworkPolicy\]"` and I noticed that all tests are failing (I actually don't know why nobody has noticed this before).  Looking into the logs I saw the following errors:

```bash
    Pod logs:
    /bin/sh: curl: not found
    /bin/sh: curl: not found
    /bin/sh: curl: not found
    /bin/sh: curl: not found
    /bin/sh: curl: not found
...    
```

which are correct because the tests using `busybox:1.29` as "client" which doesn't contain `curl`.

**What you expected to happen**:

The e2e tests for network policies should actually pass.

**How to reproduce it (as minimally and precisely as possible)**:

Just run: `kubetest --provider=skeleton --test --test_args="--ginkgo.focus=\[Feature:NetworkPolicy\]"`

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):

```bash
kubectl version                                                                                             
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"clean", BuildDate:"2019-02-04T04:48:03Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.2", GitCommit:"cff46ab41ff0bb44d8584413b598ad8360ec1def", GitTreeState:"clean", BuildDate:"2019-01-15T23:29:42Z", GoVersion:"go1.11.4", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration: kind
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

With the current fix two tests are still failing:

```bash
Summarizing 2 Failures:

[Fail] [sig-network] NetworkPolicy NetworkPolicy between server and client [It] should allow egress access on one named port [Feature:NetworkPolicy] 
/Users/jscheuermann/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/network_policy.go:398

[Fail] [sig-network] NetworkPolicy NetworkPolicy between server and client [It] should allow ingress access on one named port [Feature:NetworkPolicy] 
/Users/jscheuermann/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/network_policy.go:398

Ran 8 of 3004 Specs in 936.514 seconds
FAIL! -- 6 Passed | 2 Failed | 0 Pending | 2996 Skipped --- FAIL: TestE2E (936.59s)
```

I will take a look and fix these failing tests too.

--- a/test/e2e/network/network_policy.go
+++ b/test/e2e/network/network_policy.go
@@ -560,7 +560,7 @@ func createNetworkClientPod(f *framework.Framework, namespace *v1.Namespace, pod
 					Args: []string{
 						"/bin/sh",
 						"-c",
-						fmt.Sprintf("for i in $(seq 1 5); do wget -q -T 8 %s.%s:%d && exit 0 || sleep 1; done; exit 1",
+						fmt.Sprintf("for i in $(seq 1 5); do nc -vz -w 8 %s.%s %d && exit 0 || sleep 1; done; exit 1",
 							targetService.Name, targetService.Namespace, targetPort),
 					},
 				},

--- a/test/e2e/network/network_policy.go
+++ b/test/e2e/network/network_policy.go
@@ -17,7 +17,7 @@ limitations under the License.
 package network
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
@@ -560,7 +560,7 @@ func createNetworkClientPod(f *framework.Framework, namespace *v1.Namespace, pod
 					Args: []string{
 						"/bin/sh",
 						"-c",
-						fmt.Sprintf("for i in $(seq 1 5); do curl -s -m 8 %s.%s:%d && exit 0 || sleep 1; done; exit 1",
+						fmt.Sprintf("for i in $(seq 1 5); do wget -q -T 8 %s.%s:%d && exit 0 || sleep 1; done; exit 1",
 							targetService.Name, targetService.Namespace, targetPort),
 					},
 				},

